etl-pipeline/managers/redis.py

```diff
@@ -88,6 +88,11 @@ def set_key(
             ex=expiration_time,
         )
 
+    def any_locked(self, keys: list) -> bool:
+        keys = [f'redlock:{key}' for key in keys]
+        
+        return any(locks is not None for locks in self.client.mget(keys))
```

Comment:
could return the locked keys and log them?

Reply:
we could - but i think we want to return as quickly as possible here!